src/Solr/Forms/SearchForm.php

```diff
@@ -63,7 +63,10 @@ public function getResults()
         $request = $this->getRequestHandler()->getRequest();
 
         $searchTerms = $request->requestVar('Search');
-        $query = SearchQuery::create()->addSearchTerm($searchTerms);
+        // for pagination
+        $start = $request->requestVar('start');
```

Comment:
Perhaps we should check whether it has a value before setting it, that way we still get the default value if no argument is provided